src/main/kotlin/app/ui/UpdateRepoState.kt

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 Sourcerer Inc. All Rights Reserved.
+// Author: Anatoly Kislov (anatoly@sourcerer.io)
+
+package app.ui
+
+/**
+ * Update repositories console UI state.
+ */
+class UpdateRepoState constructor(val context: Context) : ConsoleState {
+    override fun doAction() {
+        // TODO(anatoly): Implement.
+        println("Hashing your git repositories.")
+        println("The repositories has been hashed. See the result at your "
+                + "profile page http://sourcerer.io/resume/007.")
```

Comment:
This will be very easy lose. Can you please replace this with 

... See result online on your Sourcerer profile.